Automattic/woocommerce-payments
# Code quality improvements for the Deposit Details component

#### Changes proposed in this Pull Request

This PR tidies up the code in `client/deposits/details/index.tsx` to be more consistent with the [codebase TS guidelines](https://github.com/Automattic/woocommerce-payments/tree/develop/docs/typescript) and improve code readability. It has no impact on functionality.

#### Testing instructions





* Ensure all tests pass.
* Visit a Deposit Details screen to ensure no issues with rendering. 

-------------------


**Post merge**
